src/schema.ts

```diff
@@ -131,44 +164,44 @@ export function build(data: any, opt: QueryConfig = {}): Schema {
   return new Schema(fieldSchemas);
 }
 
-export class Schema {
+export class Schema implements TableSchema {
```

Comment:
But this means that we cannot access other properties of TableSchema via Schema anyway?

Here is a small code that I experiment with:
```
interface AI {
  a: string;
  b?: number;
}

class A implements AI {
  public a: string;
}

const aa = new A();
aa.a = 'a';
aa.b = 1; // error Property b does not exist on type A
```